web/themes/joinup/prototype/sass/components/_fieldset.sass

```diff
@@ -71,6 +79,18 @@
       right: -60px
       left: -60px
 
+.fieldset--textarea
+  p
+    overflow: hidden
+  img
+    margin-bottom: 16px
+  img.align-left
```

Comment:
Also look very unstable assumption.
Also, it's not BEM at all.

Reply:
These classes comes from WYSIWYG.

Reply:
Can we move these rules into either drupal hacks or some other `editor` style sheet?